src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -52,19 +54,90 @@ private void ProcessResults()
+        /// </summary>
+        /// <param name="reader"></param>
+        private void WriteStatisticsToFile(CSVMetricReader reader)
+        {
+            //Console.WriteLine();
+            //Console.Write("[DEBUG] Press <Enter> to continue...");
+            //Console.ReadLine();
+
+            /*
+             * Generate CSV data (Probably not the most efficient way, e.g. How big can this become).
+             *  Test Name,Metric,Iterations,Average,Sample standard deviation,Minimum,Maximum
+             */
+            //var rawFilePath = $"{Configuration.RunId}-raw.csv";
+            //var rawTable = new DataTable();
+
+            var summaryFilePath = $"{Configuration.RunId}-Summary.csv";
+            var summaryTable = new DataTable();
+            var col0 = summaryTable.AddColumn("Test Name");
+            var col1 = summaryTable.AddColumn("Metric");
+            var col2 = summaryTable.AddColumn("Iterations");
+            var col3 = summaryTable.AddColumn("Average");
+            var col4 = summaryTable.AddColumn("Sample standard deviation");
+            var col5 = summaryTable.AddColumn("Minimum");
+            var col6 = summaryTable.AddColumn("Maximum");
+
+            foreach (string testCaseName in reader.TestCases)
+            {
+                List<Dictionary<string, double>> iterations = reader.GetValues(testCaseName);
+
+                var measurements = new Dictionary<string, List<double>>();
+                foreach (var dict in iterations)
+                {
+                    foreach (var pair in dict)
+                    {
+                        if (!measurements.ContainsKey(pair.Key))
+                            measurements[pair.Key] = new List<double>();
+                        measurements[pair.Key].Add(pair.Value);
+                    }
+                }
+
+                foreach (var measurement in measurements)
+                {
+                    var metric = measurement.Key;
+                    var count = measurement.Value.Count;
+                    var avg = measurement.Value.Average();
+                    var stdev_s = Math.Sqrt(measurement.Value.Sum(x => Math.Pow(x - avg, 2)) / (measurement.Value.Count - 1));
+                    var max = measurement.Value.Max();
+                    var min = measurement.Value.Min();
+
+                    var r = summaryTable.AppendRow();
+                    r[col0] = testCaseName;
+                    r[col1] = metric;
+                    r[col2] = count.ToString();
+                    r[col3] = avg.ToString();
+                    r[col4] = stdev_s.ToString();
+                    r[col5] = min.ToString();
+                    r[col6] = max.ToString();
+
+                    //Console.WriteLine($"{testCaseName},{metric},{count},{avg},{stdev_s},{min},{max}");
+                }
+            }
+
+            summaryTable.WriteToCSV(summaryFilePath);
+            Console.WriteLine($"Created \"{summaryFilePath}\"");
```

Comment:
Can you change this to "Results written to "<path>""?  Makes it clear to the user what's in here.